crates/cheatcodes/src/inspector.rs

```diff
@@ -1250,6 +1254,37 @@ impl Cheatcodes {
             return None;
         }
 
+        let mut create2_factory_deps = Vec::new();
```

Comment:
I'd make it a bit more explicit what these are, because create2 is not just the factory

Reply:
These must be treated as simply `factory_deps` to any arbitrary call. Nothing related to CREATE2.